src/Build/BackEnd/Components/Caching/ResultsCacheWithOverride.cs

```diff
@@ -77,14 +77,12 @@ public BuildResult GetResultsForConfiguration(int configurationId)
             BuildRequest request,
             List<string> configInitialTargets,
             List<string> configDefaultTargets,
-            List<string> additionalTargetsToCheckForOverallResult,
             bool skippedResultsDoNotCauseCacheMiss)
         {
             var overrideRequest = _override.SatisfyRequest(
```

Comment:
This project actually has a well-defined coding style (linked from the docs, although the link is broken -- will fix that momentarily). This is what it has to say for `var`:

`
We only use var when it's obvious what the variable type is (e.g. var stream = new FileStream(...) not var stream = OpenStandardInput()).
`

I'll go over my changes and make sure I'm following this. Thanks!

Reply:
I have replaced two concrete types with `var` in accordance with the coding guidelines and to match what `GetAfterTargetsForDefaultTargets` had.

Also added a commit fixing the broken doc links. Hope we're not too crazy about strictly keeping such changes in separate PRs.

Reply:
Consistent coding practice is good; the problem is determining what that coding practice should be. cdmihai is in favor of var even when it's very not obvious to me because it forces him to use better variable names. I'm in favor of concrete types everywhere with very rare exceptions when we can't figure out the type until runtime. It's clearer if you want to know what type something is without looking at more than a single spot, and it helps with VS's "go to definition," which is sometimes confused by imprecision and takes me somewhere useless instead of what I want.
I suggested talking about this at some point at a time when it was bothering me regularly, but we never had it, and it isn't bothering me as often right now. I'd still be happy to have such a discussion.

Reply:
Roslyn now supports formalizing style questions like this in a `.editorconfig`, and then you can apply fixes across the entire source tree. I've been meaning to put together my set of proposed ones but haven't gotten around to it.